### DISCUSSION TITLE
Trying to get GCAM's latest releases linked to Obtainium - [use of HTML sources]

### DISCUSSION THREAD

**Discussion by n-cash** [ImranR98/Obtainium#1831] [Q&A] [answered]
Hey everyone, just like the title says. I'm trying to get the latest GCAM release extracted from Celso Azevedos website.

In case anyone doesn't have a clue of what a GCAM is, it's short for "Google Camera". The camera app developed by Google for its Pixel devices which has many advanced camera features such as HDR+, Night Sight, etc... [Celso Azevedo](https://www.celsoazevedo.com/files/android/google-camera/) is known for hosting a collection of modified versions of this app, often referred to as GCam ports. Which are adapted by various developers to work on a wide range of Android devices, and not just Google's Pixel phones, -see [this comparison of Stock Camera on regular phones vs GCam Ports, if you're interested.](https://www.celsoazevedo.com/files/android/google-camera/examples/). There's a clear difference on the picture quality that is achievable even with a budget phone, if using this Google Camera instead of Stock.

The thing is, i've poked around with the "Update Alerts", that this site provides. These come in the form of: [Telegram messages](https://t.me/googlecameraport), an [rss feed](https://www.celsoazevedo.com/files/android/google-camera/dev-feed.xml), [twitter](https://x.com/GcamPort) AND [mastodon](https://mastodon.social/@gcam) posts. Yet i still can't find a way to get Obtainium to navigate inside of these links and extract the newer versions for my specific BSG Developed GCam.

¿Has anyone come across something similar to this, where the app isn't on the mainstream Foss channels that Obtainium is meant for?
**I know that what I need to do is set up these URLs inside of Obtainiums HTML source tab, but don't know to do that, in a way that works.**

**Comment by ImranR98** [comment, marked as answer]
This might be helpful: https://github.com/ImranR98/Obtainium/issues/1182#issuecomment-1873155267

**Reply by n-cash** [reply]
Yes, this is it. Thanks a lot!

**Reply by fuzi0nz** [reply]
Hey is this still working for you? I keep getting 404 error.
